python/orca/src/bigdl/orca/learn/pytorch/core/modelIO.py

```diff
@@ -14,10 +14,15 @@
 # limitations under the License.
 #
 
-from abc import ABCMeta, abstractmethod
+import os
```

Comment:
Shall we avoid these global dependencies?

Reply:
import os globally should be ok😂
but if some package is only used once, then it is better practice to make it local import